### DISCUSSION TITLE
Xbox button prompts

### DISCUSSION THREAD

**Discussion by jayshai** [open-goal/jak-project#2965] [Ideas]
It'd be cool if you could add the option for Xbox button prompts to the pause menu and where the button icons are used.

**Comment by Zedb0T** [comment]
Xbox prompts have been done by Nax : https://cdn.discordapp.com/attachments/988955638946234408/988958616935620608/texture_replacements.7z

**Reply by Zedb0T** [reply]
![image](https://github.com/open-goal/jak-project/assets/89345505/58796ef3-b4f5-4d79-bcf2-c51cb7e7b66c)
![image](https://github.com/open-goal/jak-project/assets/89345505/d20b2eb3-ee6e-49b0-966c-092102e56166)

**Comment by Joseph-DiGiovanni** [comment]
Link no longer works @Zedb0T 

Here's some new ones I found: [Jak1](https://cdn.discordapp.com/attachments/1170416696243257404/1328188621580144701/Jak1_Xbox_Buttons.zip?ex=6957d5cb&is=6956844b&hm=732bf39fef1c65699d381e4d585989ba6294d80b5bef77a04f67e6f222398aa0&) & [Jak2](https://cdn.discordapp.com/attachments/1170416696243257404/1328188621874008075/Jak2_Xbox_Buttons.zip?ex=6957d5cc&is=6956844c&hm=8ae430406b9806b848af642a0fa0cda098ae38e29abf01882aa3d4aa674f1d8e&) via [this message](https://discord.com/channels/756287461377703987/1170416696243257404/1328188622180188253)

**Reply by bmalinowski89** [reply]
@Zedb0T @Joseph-DiGiovanni this link is now inactive. can we not share this somewhere not in a discord? I'd like to update my button prompts.

**Reply by dallmeyer** [reply]
check nexus mods
